src/verifier.cc

```diff
@@ -647,8 +647,10 @@ Result Verifier::ProbeSSBO(const ProbeSSBOCommand* command,
                                 "while expected data is not");
   }
 
-  const auto& datum_type = command->GetDatumType();
-  size_t bytes_per_elem = datum_type.ElementSizeInBytes();
+  auto* fmt = command->GetFormat();
+  // TODO(dsinclair): This assumes that all components are the same number
+  // of bits which may not alway shold.
+  size_t bytes_per_elem = fmt->GetComponents()[0].num_bits / 8;
```

Comment:
Should the '8' number litteral be replaced by a const?

Reply:
done